### DISCUSSION TITLE
can we directly migrate 2.24.1 to 5.1.3 its possible

### DISCUSSION THREAD

**Discussion by Hariprakash-Jawahar** [ghiscoding/Angular-Slickgrid#1066] [Q&A] [answered]
Hi i'm implement angular slickgrid 2.24.1 now needs to update to latest can we directly update to 5.1.3 its possible ?

**Comment by ghiscoding** [comment, marked as answer]
yes but you still need to follow all the [migration guides](https://github.com/ghiscoding/angular-slickgrid/wiki/Migration-to-3.x) and apply necessary (breaking) changes required between each versions, you also need to verify Angular requirements. Ideally, it would be better to upgrade 1 version at a time, following [Anglar Compatibility](https://github.com/ghiscoding/Angular-Slickgrid#angular-compatibility) and confirm that it works
